examples/hello-world/helloworld.yaml

```diff
@@ -0,0 +1,141 @@
+      properties:
+        external_network: external
+        create_if_missing: true
+        resource_id: aria_helloworld_rtr
+
+    subnet:
+      type: aria.openstack.nodes.Subnet
+      properties:
+        resource_id: aria_helloworld_subnet
+        create_if_missing: true
+        subnet:
+          cidr: 172.16.0.0/16
+          ip_version: 4
+      requirements:
+        - router: router
+        - network: network
+
+    port:
+      type: aria.openstack.nodes.Port
+      properties:
+        create_if_missing: true
+        resource_id: aria_helloworld_port
+      requirements:
+        - security_group: security_group
+        - floating_ip: virtual_ip
+        - subnet: subnet
+        - network: network
+
+    virtual_ip:
+      type: aria.openstack.nodes.FloatingIP
+      properties:
+        resource_id: aria_helloworld_floatingip
+        create_if_missing: true
+        floatingip:
+          floating_network_id: 6751cb30-0aef-4d7e-94c3-ee2a09e705eb # { get_input: external_network_id }
+
+    security_group:
+      type: aria.openstack.nodes.SecurityGroup
+      properties:
+        create_if_missing: true
+        resource_id: aria_helloworld_sg
+        rules:
+          - remote_ip_prefix: 0.0.0.0/0
+            port: 8080
+          - port: 22
+            remote_ip_prefix: 0.0.0.0/0
+          - port: 5985
+            remote_ip_prefix: 0.0.0.0/0
+
+    keypair:
+      type: aria.openstack.nodes.KeyPair
+      properties:
+        create_if_missing: true
+        resource_id: aria_helloworld_kp
+        private_key_path: /tmp/aria-helloworld-os-keypair
+
+    vm:
+      type: aria.openstack.nodes.Server
+      properties:
+        image: ubuntu-14.04
+        flavor: dc1.8x16
+        create_if_missing: true
+        resource_id: aria_helloworld_vm
+        management_network_name: aria_helloworld_network
+      requirements:
+        - floating_ip: virtual_ip
+        - security_group: security_group
+        - key_pair: keypair
+        - port: port
+
+    # Currently missing the host_string. thus no real executions can run until the intrinsic
+    # functions are implemented
+    web_app:
+      type: web_app
+      properties:
+        port: 8080
+      requirements:
+        - host: vm
+      interfaces:
+        Standard:
+          configure:
+            implementation:
+              primary: scripts/configure.sh
+              dependencies:
+                - ssh.user > ubuntu
+                - ssh.key_filename > /tmp/aria-helloworld-os-keypair
+          start:
+            implementation:
+              primary: scripts/start.sh
+              dependencies:
+                - ssh.user > ubuntu
+                - ssh.key_filename > /tmp/aria-helloworld-os-keypair
+          stop:
+            implementation:
+              primary: scripts/stop.sh
+              dependencies:
+                - ssh.user > ubuntu
+                - ssh.key_filename > /tmp/aria-helloworld-os-keypair
+
+  policies:
```

Comment:
consider moving policy to plugin.yaml, even if it means having `topology_templates` section in plugin.yaml
check if it works :)